onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -417,9 +417,9 @@ TEST(CApiTest, instant_op_handler) {
   std::vector<int64_t> expected_dims_y = {3, 2};
   std::vector<float> expected_values_y = {2.0f, 4.0f, 6.0f, 8.0f, 10.0f, 12.0f};
 
-  InstantCustomOp instant_op{onnxruntime::kCpuExecutionProvider, nullptr};
+  StandaloneCustomOp standalone_op{onnxruntime::kCpuExecutionProvider, nullptr};
   Ort::CustomOpDomain custom_op_domain("");
```

Comment:
If the customer has an existing binary code, that calls this API, and they do, should we not make sure that it works as before? I think we need to change the struct version, is the same entry in the OrtApi struct does not map to the same function.

Reply:
Same as above - before it is made public in 1.12, we could continue to evolve the feature by some API changes.
The fact is,  the change here is rightly required by the customer who would officially consume it on 1.12.